indico/util/string.py

```diff
@@ -522,10 +615,11 @@ class RichMarkup(Markup):
     it in a jinja template.
     """
 
-    __slots__ = ('_preformatted',)
+    __slots__ = ('_preformatted', '_linker')
 
-    def __new__(cls, content='', preformatted=None):
+    def __new__(cls, content: str = '', preformatted: t.Optional[bool] = None, linker: t.Optional[t.Callable] = None):
```

Comment:
Am I blind or is this not used anywhere (with the new arg you added)?

Reply:
Indeed. Forgot to delete this change after I changed strategy. Thanks!

Reply:
I'll leave the types as a bonus, though 😆